gwlfe/datamodel.py

```diff
@@ -105,6 +107,31 @@ def defaults(self):
             'CSNDevType': 'None',
         }
 
+    def fill_date_guides(self):
+        model = self.__dict__
+        if 'WxYrBeg' in model and 'WxYrEnd' in model:
+            begyear = model['WxYrBeg']
+            endyear = model['WxYrEnd']
+            year_range = endyear - begyear + 1
+            month_abbr = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+                          'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+
+            if 'DaysMonth' not in model:
+                model['DaysMonth'] = np.ones((year_range, 12),
```

Comment:
I don't think it's terribly important to initialize each month to have 31 days when it gets assigned the correct number in the next loop. `np.zeros` may be good enough.